### DISCUSSION TITLE
Development setup?

### DISCUSSION THREAD

**Discussion by johnniehard** [vkurko/calendar#237] [General]
I'm wondering how to go about setting this up for local development as I didn't find anything about it in the README.

**Comment by vkurko** [comment]
Please use [this](https://github.com/vkurko/calendar/discussions/59#discussioncomment-3623690) as a reference.

**Reply by johnniehard** [reply]
Thanks!

**Comment by gavinndev** [comment]


**Reply by vkurko** [reply]
`npm run build`
